include/functions.h

```diff
@@ -605,15 +605,15 @@ u16 SSNodeList_GetNextNodeIdx(SSNodeList* this);
 s32 DynaPoly_IsBgIdBgActor(s32 bgId);
 void DynaPoly_Init(PlayState* play, DynaCollisionContext* dyna);
 void DynaPoly_Alloc(PlayState* play, DynaCollisionContext* dyna);
-void func_8003EBF8(PlayState* play, DynaCollisionContext* dyna, s32 bgId);
-void func_8003EC50(PlayState* play, DynaCollisionContext* dyna, s32 bgId);
-void func_8003ECA8(PlayState* play, DynaCollisionContext* dyna, s32 bgId);
+void DynaPoly_SetBgIdDisabled(PlayState* play, DynaCollisionContext* dyna, s32 bgId);
+void DynaPoly_UnsetBgIdDisabled(PlayState* play, DynaCollisionContext* dyna, s32 bgId);
+void DynaPoly_SetBgIdSkipCeilings(PlayState* play, DynaCollisionContext* dyna, s32 bgId);
```

Comment:
it's inspired from `DynaPoly_IsBgIdBgActor` and naming like `DynaPoly_SetBgActorPrevTransform`

What would be a better way to name these?

Reply:
`DynaPoly_IsBgIdBgActor` is specifically performing a check on bgId (in this case, if the id maps to a BgActor instead of i.e. -1 or BG_SCENE), hence it's name.